charts/geonode/templates/nginx/nginx-conf.yaml

```diff
@@ -119,6 +106,22 @@ data:
         proxy_http_version 1.1;
       }
 
+
+      {{if (eq .Values.geonode.ingress.enabled false) }}
+      # GeoServer
+      location /geoserver {
+        client_max_body_size {{ .Values.nginx.maxClientBodySize }};
+
+{{- if .Values.nginx.external_cors.enabled }}
```

Comment:
You may consider [removing CORS config at nginx](https://github.com/GeoNode/geonode-docker/issues/42) completely, just in favor of configuring CORS via [django-cors-headers](https://pypi.org/project/django-cors-headers/).

Reply:
@AlexGacon any opinion on that from your side?

Reply:
It could be a solution, even if it implies having custom code on the geonode side if required (but since we now support GeoNode projects, it sounds possible)